extensions/wikia/FounderEmails/events/FounderEmailsRegisterEvent.class.php

```diff
@@ -1,101 +1,52 @@
+		return true;
 	}
 
 	public function process( Array $events ) {
-		global $wgEnableAnswers, $wgCityId;
-		wfProfileIn( __METHOD__ );
-
-		if ( $this->isThresholdMet( count( $events ) ) ) {
-			// get just one event when we have more... for now, just randomly
-			$eventData = $events[rand( 0, count( $events ) -1 )];
 
-			$founderEmailObj = FounderEmails::getInstance();
-			$wikiService = ( new WikiService );
-			$user_ids = $wikiService->getWikiAdminIds();
-			$foundingWiki = WikiFactory::getWikiById( $wgCityId );
-
-			$emailParams = array(
-				'$EDITORNAME' => $eventData['data']['userName'],
-				'$EDITORPAGEURL' => $eventData['data']['userPageUrl'],
-				'$EDITORTALKPAGEURL' => $eventData['data']['userTalkPageUrl'],
-				'$WIKINAME' => $foundingWiki->city_title,
-				'$WIKIURL' => $foundingWiki->city_url,
-			);
-			$wikiType = !empty( $wgEnableAnswers ) ? '-answers' : '';
-
-			foreach ( $user_ids as $user_id ) {
-				$user = User::newFromId( $user_id );
-
-				// skip if not enable
-				if ( !$this->enabled( $wgCityId, $user ) ) {
-					continue;
-				}
-				self::addParamsUser( $wgCityId, $user->getName(), $emailParams );
-
-				$langCode = $user->getOption( 'language' );
-				$mailSubject = strtr( wfMsgExt( 'founderemails' . $wikiType . '-email-user-registered-subject', array( 'content' ) ), $emailParams );
-				$mailBody = strtr( wfMsgExt( 'founderemails' . $wikiType . '-email-user-registered-body', array( 'content' ) ), $emailParams );
+		if ( !$this->isThresholdMet( count( $events ) ) ) {
+			return false;
+		}
 
-				if ( empty( $wgEnableAnswers ) ) { // FounderEmailv2.1
-					$links = array(
-						'$EDITORNAME' => $emailParams['$EDITORPAGEURL'],
-						'$WIKINAME' => $emailParams['$WIKIURL'],
-					);
-					$mailBodyHTML = F::app()->renderView( 'FounderEmails', 'GeneralUpdate', array_merge( $emailParams, array( 'language' => 'en', 'type' => 'user-registered' ) ) );
-					$mailBodyHTML = strtr( $mailBodyHTML, FounderEmails::addLink( $emailParams, $links ) );
-				} else {
-					$mailBodyHTML = $this->getLocalizedMsg( 'founderemails' . $wikiType . '-email-user-registered-body-HTML', $emailParams );
-				}
+		// This event is triggered when a new user registers, so we
+		// know there's only one event in the events array
+		$eventData = $events[0]['data'];
+		$emailParams = [
+			'currentUser' => $eventData['newUserName'],
+		];
 
-				$mailCategory = FounderEmailsEvent::CATEGORY_REGISTERED . ( !empty( $langCode ) && $langCode == 'en' ? 'EN' : 'INT' );
+		foreach ( ( new WikiService )->getWikiAdminIds() as $adminId ) {
+			$admin = User::newFromId( $adminId );
+			$emailParams['targetUser'] = $admin->getName();
 
-				wfProfileOut( __METHOD__ );
-				$founderEmailObj->notifyFounder( $user, $this, $mailSubject, $mailBody, $mailBodyHTML, $wgCityId, $mailCategory );
-			}
-			return true;
+			F::app()->sendRequest( self::EMAIL_CONTROLLER, 'handle', $emailParams );
 		}
 
-		wfProfileOut( __METHOD__ );
-		return false;
+		return true;
```

Comment:
Nope. You'll see that before editing this method it returned in a few different places. It still returns false if it's an anwersWiki, but otherwise the 3 other checks about whether we send the email or not got moved into the Email extension.